src/fiatconnect/index.ts

```diff
@@ -191,3 +193,12 @@ export function getObfuscatedAccountNumber(accountNumber: string): string {
   const digitsToReveal = Math.max(0, Math.min(accountNumber.length - 3, 4))
   return digitsToReveal > 0 ? '...' + accountNumber.slice(-digitsToReveal) : ''
 }
+
+export async function doTransferOut(fiatConnectQuote: FiatConnectQuote, fiatAccountId: string) {
```

Comment:
any reason you chose to abstract this function here? maybe there are future plans to use this elsewhere that i'm not aware of :) was just thinking that this would be better as a generator function in fiatconnect/saga file since it is being used only in a saga for now?

Reply:
good question! i don't think there are benefits actually (correct me if i'm wrong @jeanregisser ?) i was only mentioning it from the perspective that async functions that are only called by sagas can be thought of as sagas too so it's more clear to keep them in one place/format :)